.gitignore

```diff
@@ -1,5 +1,105 @@
-# Python temp/bin files
-*.py[co]
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
 
-# ViM temp files
-.*.sw?
```

Comment:
A czemu wyciąłeś mi tymczasowe pliki ViM'a?

Reply:
Do dalszych linijek, potrzebujemy tego wszystkiego?
Np. Flask, Django itp.

Boje się, że możemy za łatwo odciąć coś ważnego. Takie nazwy jak dist, instance, target nie wydają się tylko i wyłącznie nazwami roboczymi, Owszem, w konkretnych projektach tak, ale we wszystkich?

Może jednak będziemy powoli rozszerzać? Jak nam `git status` pokaże śmieci, to dodamy wtedy do `.gitignore`.

Reply:
To standardowy plik gitignore dla pythona odcinający zbędne pliki w razie gdyby się tam przypadkiem znalazły. A potrzebne są ViM w repo? I pliki zip nie znalazły się przypadkiem tutaj, łatwiej sprawdzać wtyczki gdy zainstaluje się je z pliku zip. Jeśli Kodi jest czyste i wkleisz bezpośrednio folder z wtyczką to nie zostanie ona aktywowana automatycznie, zipami ułatwiam sobie życie. Ale jeśli ich nie chcemy to można dodać do gitignore, mogę to sobie robić lokalnie :)